### DISCUSSION TITLE
Why dashboard instance glare widget not working in mobile view?

### DISCUSSION THREAD

**Discussion by faizalsb** [firefly-iii/firefly-iii#4377] [General]
I am running Firefly III version 5.4.6

**Description**
Instant glare widget ("Balance", "bills to pay", "spent" and "net worth")
not appear if browser turn to mobile view.

**Extra info**
 

**Bonus points**

- I have read the FAQ at https://bit.ly/FF3-FAQ
- I added a screenshot
- I added log files 
- I was able to replicate the issue on the demo site.

**Comment by JC5** [comment]
That's correct. They're hidden om small screens to save space.

**Comment by truppelito** [comment]
I was going to write an issue myself but found this first. I would like to +1 this. I understand the idea of saving space, but these four values are, for me, the best and clearest way to keep my finances in check at a glance. I would love it if they were available on my phone (which is where I use Firefly III the most).
